cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1543,10 +1543,12 @@ function DownloadAndInstall-NodeProblemDetector {
 #   CA_CERT
 #   NODE_PROBLEM_DETECTOR_TOKEN
 function Create-NodeProblemDetectorKubeConfig {
-  if (-not [string]::IsNullOrEmpty(${env:NODEPROBLEMDETECTOR_KUBECONFIG_FILE})) {
-    Create-Kubeconfig -Name 'node-problem-detector' `
-      -Path ${env:NODEPROBLEMDETECTOR_KUBECONFIG_FILE} `
-      -Token ${kube_env}['NODE_PROBLEM_DETECTOR_TOKEN']
+  if ("${env:ENABLE_NODE_PROBLEM_DETECTOR}" -eq "standalone") {
+    if (-not [string]::IsNullOrEmpty(${env:NODEPROBLEMDETECTOR_KUBECONFIG_FILE})) {
```

Comment:
Can we match the logic in Linux nodes, that is, if `NODE_PROBLEM_DETECTOR_TOKEN` is empty, then use kubelet kubeconfig?

https://github.com/kubernetes/kubernetes/blob/367ef6ffe13e16a14bbc3623f450692e57066fc3/cluster/gce/gci/configure-helper.sh#L3598-L3608

Reply:
> Can we match the logic in Linux nodes, that is, if `NODE_PROBLEM_DETECTOR_TOKEN` is empty, then use kubelet kubeconfig?
> 
> https://github.com/kubernetes/kubernetes/blob/367ef6ffe13e16a14bbc3623f450692e57066fc3/cluster/gce/gci/configure-helper.sh#L3598-L3608


@AnishShah Hi mate, I have already amended these code to match the logic of Linux nodes, but it seems like I don't have permission to push code directly to this PR. Can I create a new PR for the changes?

Reply:
Yes you can continue with a new PR if you want, one reason I didn't continue is that I didn't get time to test this with a GCE cluster following these instructions https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/README-GCE-Windows-kube-up.md

Reply:
I matched the logic to be the same as linux but I didn't test that codepath since we set the env var `ENABLE_NODE_PROBLEM_DETECTOR=none` which bypasses the logic above.